src/javascript/hooks/graphql/useGetSitemapUrl.js

```diff
@@ -0,0 +1,33 @@
+import {useEffect, useState} from 'react';
+import {useLazyQuery} from '@apollo/react-hooks';
+import {GetSitemapUrl} from '../../components/gqlQueries';
+
+const useGetSitemapUrl = siteKey => {
```

Comment:
Not a big deal in this case but it looks like for this one we could get away with a standard useQuery({variables : {siteKey}}), it should be smart enough to recognize that query needs to run again, unless I misunderstand something.

Reply:
Looks like no difference between lazyQuery and standard useQuery. Switched to useQuery instead, thanks.